AssetOptionsBinder.cs

```diff
@@ -112,6 +112,22 @@ internal class AssetOptionsBinder : BinderBase<AssetOptions>
             () => DefaultBatchSize,
             description: @"Batch size for parallel processing.");
 
+        private readonly Option<bool> _encryptContent = new (
+            aliases: new[] { "-e", "--encrypt-content" },
+            () => false,
+            description: "Encrypt the content  using CENC"
+            );
+
+        private readonly Option<Uri?> _keyVaultUri = new (
+            aliases: new[] { "--key-vault-uri" },
+            description: "The key vault to store encryption keys."
+            );
+
+        private readonly Option<string?> _keyUri = new(
```

Comment:
I think we don't allow user to specify the keyURI, the key URI must be relative uri `/.clearkeys`. The uri path is unconfigurable in proxy server.

Reply:
Instead of hardcoding in the tool, the user can pass /.clearkeys...  we need to pass it because we need to fill in the key id in the URL. (which is template)  some something like ?key_id=${KeyId} will be expanded to ?key_id=acutalkeyid

Reply:
I'm a little confused, is the option provided by the user or by the migration tool?

Reply:
The option is provided by the customer to the migration tool (and the tool expanded the URL to include the key id and passes it to shaka packager).  The tool is not limited to be used by our own proxy service. It can be used with any other service that servs the keys (by giving full URL)

Reply:
I don't think the tool can be used by other servers. The key format you used is hex string which is not natively supported by neither DASH nor HLS. DASH uses base64 encoded key, and HLS uses raw bytes key.

At least, I think we should make this option as optional. By default, the migration tool will generate the right license url for user. Otherwise, user doesn't know the url actually doesn't work until they deploy all things. And they need to regenerate to fix this issue, it will waste a lot of time.